### DISCUSSION TITLE
postfix-monitoring/

### DISCUSSION THREAD

**Discussion by giscus** [netdata/blog#347] [General]
# postfix-monitoring/

Monitoring and troubleshooting Postfix.

https://blog.netdata.cloud/postfix-monitoring/

**Comment by RahmatAliMalik5** [comment]
Hi,

Is there a way to get other parameters as well, like "sent emails", "received emails" etc? qemails shows on mails in the queue. But I want to see how many mails are going out of my server. Thanks
